src/sql/src/plan/statement/ddl.rs

```diff
@@ -1458,88 +1481,10 @@ pub fn plan_create_views(
-                                    expr: Box::new(Expr::Identifier(vec![Ident::new("row_data")])),
-                                    positions: vec![SubscriptPosition {
-                                        start: Some(Expr::Value(Value::Number(
-                                            // LIST is one based
-                                            (i + 1).to_string(),
-                                        ))),
-                                        end: None,
-                                        explicit_slice: false,
-                                    }],
-                                }),
-                                data_type,
-                            },
-                            alias: Some(Ident::new(column_name.as_str())),
-                        });
-                    }
-                    let query = Query {
-                        ctes: vec![],
-                        body: SetExpr::Select(Box::new(Select {
-                            distinct: None,
-                            projection,
-                            from: vec![TableWithJoins {
-                                relation: TableFactor::Table {
-                                    name: source_name.clone(),
-                                    alias: None,
-                                },
-                                joins: vec![],
-                            }],
-                            selection: Some(Expr::Op {
-                                op: Op::bare("="),
-                                expr1: Box::new(Expr::Identifier(vec![Ident::new("table")])),
-                                expr2: Some(Box::new(Expr::Value(Value::String(
-                                    table.to_string(),
-                                )))),
-                            }),
-                            group_by: vec![],
-                            having: None,
-                            options: vec![],
-                        })),
-                        order_by: vec![],
-                        limit: None,
-                        offset: None,
-                    };
-                    let mut viewdef = ViewDefinition {
-                        name: UnresolvedObjectName::unqualified(table),
-                        columns: columns
-                            .iter_names()
-                            .map(|name| Ident::from(name.as_str()))
-                            .collect(),
-                        query,
-                    };
-                    plan_view(scx, &mut viewdef, &Params::empty(), temporary)
-                })
-                .collect::<Result<Vec<_>, _>>()?;
-            Ok(Plan::CreateViews(CreateViewsPlan {
-                views,
-                if_not_exists: if_exists == IfExistsBehavior::Skip,
-            }))
-        }
-        connection => sql_bail!("cannot generate views from {} sources", connection.name()),
-    }
+    sql_bail!(
```

Comment:
If we land this by Tuesday can we just remove this entirely?

Reply:
This is my next PR once this lands